### DISCUSSION TITLE
Combining Spatial Transcriptomics with IBEX imaging

### DISCUSSION THREAD

**Discussion by anita-gola** [IBEXImagingCommunity/ibex_imaging_knowledge_base#186] [General]
Dear all, 
We are currently interesting in performing detailed characterization of mouse skin tissue using IBEX in combination with Spatial Transcriptomic techniques (particularly Visium HD). 
We are wondering what would be the best tissue prep to use: FFPE vs fresh-frozen (which would be our preferred method). We are aware that while fresh-frozen section are available for Visium/Xenium, they are currently still in the pipeline for Visium HD. Does anyone have thoughts/experiences they'd like to share on the matter?
Thanks again for everything!
Anita

**Comment by radtkea** [comment]
Hi Anita,

Thanks very much for your question! Very excited for your project! 

I haven't worked with Visium HD, but based on their literature it seems that they only support FFPE tissues. See below.
[What tissue preservation methods are compatible with the Visium platform?](https://kb.10xgenomics.com/hc/en-us/articles/360036305111-What-tissue-preservation-methods-are-compatible-with-the-Visium-Platform)
[Visium HD Blog post.](https://www.10xgenomics.com/blog/your-introduction-to-visium-hd-spatial-biology-in-high-definition)

The [Visium HD protocol ](https://www.10xgenomics.com/support/spatial-gene-expression-hd/documentation/steps/tissue-prep/visium-hd-ffpe-tissue-preparation-handbook) provided by 10X is very detailed. They recommend 3-10 um sections for best performance and additionally detail how to do antibody labeling in later steps. 

[My colleagues](https://www.protocols.io/view/ffpe-tissue-processing-for-multimodal-imaging-assa-dbmp2k5n?step=1) recently integrated Xenium and Phenocycler imaging in a single tissue section. They performed a gentle antigen retrieval step (80C) before Xenium. Following Xenium, they performed another antigen retrieval step (90-110C) for antibody-based imaging with a Phenocycler. While this is not Visium HD, I found it quite interesting that they had to do additional antigen retrieval prior to multiplexed imaging with antibodies.

In conclusion, I think you will might be limited to FFPE samples. In this case, you will need to adjust your IBEX panel. Do you have a list of markers you are interested in for the skin? We can help you build a panel.

I will share your question with some experts too:) @nadavyayon would you be able to add your input here? Could you also share with Omer?

Best of luck and thanks for asking in the open!
A

**Reply by nadavyayon** [reply]
Hey Anita. 

We have been working with visiumHD recently and developed a pipeline for nuclei segmentation and binning https://github.com/Teichlab/bin2cell which might be useful for you to align the images from the different platforms. 

I think people are now trying to combine protein imaging and visiumHD but opposed to xenium these would all have to be performed before visiumHD as the process is destructive of the tissue. 

I think 10x are currently working on a fresh frozen solution for visiumHD but there is no official release for that. However, i assume visiumHD would work on tissues that were fixed frozen as is recommended for ibex. We have tested in-house the use of pax gene with visiumSD ffpe and it works but the binding probability of the probes is shifted and I assume will be the case as well for fixed frozen. 10x have calibrated the probe concentration for ffpe, so this is expected.  

If you would want to test I would recommend using visium SD which uses the same probe system and is much cheaper before attempting to do this on visiumHD. 

Good luck and keep us posted!!

Nadav

P.s. one comment about capture of xenium and visiumHD. Even the 5k xenium misses many important markers for thymus such as 95% of the KRT genes (for TECs) as well as LMNA. But also visiumHD is lacking several important genes such as XIST and CD8B. So nothing is perfect!!

**Reply by radtkea** [reply]
Excellent response Nadav! Thank you so much. Warmest congrats on the software and [preprint](https://www.biorxiv.org/content/10.1101/2024.06.19.599766v1.full)! Go team!!!

Hope this helps @anita-gola! I love being your friends:)

**Reply by anita-gola** [reply]
Thanks so much to both- greatly appreciate! I will also share as well that in our conversations with 10x reps we have been told that they are working on demonstrated protocols for both fresh and fixed frozen tissues for Visium HD, and they expect to have those by the end of Q3 2024.
